cms/partners/map-companyname-veritag-pte-ltd-description-tracking-and-trace-food-and-arts-using-patented-dual-qr-tags-blockchain-and-nft-technology-supported-by-documents-security-using-tradetrust-websitelink-www-veritag-com-logo-static-upl.md

```diff
@@ -0,0 +1,9 @@
+---
+companyName: veriTAG Pte Ltd
+description: Tracking and trace food and arts using patented dual QR tags,
+  blockchain and NFT technology, supported by documents security using
+  TradeTrust.
+websiteLink: www.veritag.com
```

Comment:
www.veritag.com not linking out correctly

Reply:
ok sure.